src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <DashboardCard width="full">
+  <DashboardCard width="two-thirds">
```

Comment:
Changes in this file don't seem to be related to this PR?

Reply:
What are these changes for?

Reply:
ideally, it would have been a different PR, it is just to make the buoy image on it's own card

Reply:
Ahh okay!